### ISSUE TITLE
Specifying defaultTrace method with multiple arguments

### ISSUE THREAD

**Issue by fxnn** [alfredxiao/jackplay#6]
It's impossible to specify a method in `defaultTrace` that has more than one arguments, as these are separated using a comma, which however is used to separate arguments:

    -javaagent:jackplay-agent.jar="defaultTrace=com.example.myMethod(java.lang.String,java.lang.String)"

yields

```
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:386)
	at sun.instrument.InstrumentationImpl.loadClassAndCallPremain(InstrumentationImpl.java:401)
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.substring(String.java:1967)
	at jackplay.bootstrap.Options.parseArguments(Options.java:48)
	at jackplay.bootstrap.Options.optionsMergedWithDefaults(Options.java:39)
	at jackplay.Theatre.premain(Theatre.java:35)
	... 6 more
FATAL ERROR in native method: processing of -javaagent failed
Exception in thread "main" 
```

**Comment by alfredxiao** [comment]
Could you give details of your setup, especially JDK version and OS platform?

**Comment by fxnn** [comment]
The bug seems to be independent of JDK and OS versions. Options are simply split by comma (cf. [Options.java#45](https://github.com/alfredxiao/jackplay/blob/master/jackplay/src/jackplay/bootstrap/Options.java#L45) without regarding parantheses, so there's no way for jackplay to process an option containing a comma. However, this is required when using methods with multiple arguments.

**Comment by alfredxiao** [comment]
Apologize that I should have read your message in detail. Yeah, you are correct that this is an apparent bug in the use of comma. Easy fix is to use ";" as option separator and keep "," used for separating arguments when specifying a method in defaultTrace thne. Thanks for pointing this out.

**Comment by alfredxiao** [comment]
Released 0.10.0 with the fix where options are separated by ";" while method arguments are separated by ",".
